packages/web-components/src/components/card-group/__stories__/card-group.stories.ts

```diff
@@ -37,7 +38,10 @@ const cardRandomPhrase = () => {
     'Te sint disputando pri, at his aliquip corrumpit',
   ];
 
-  const randomSampleText = phraseArray[Math.floor(Math.random() * phraseArray.length)];
+  const randomSampleText = inPercy()
+    ? 'Lorem ipsum dolor sit amet, consectetur'
+    : phraseArray[Math.floor(Math.random() * phraseArray.length)];
+
```

Comment:
Thanks @NSDrowned !

I'm wondering if we should still pass in varying text sizes, but keep them static, just so we can also capture varying heights in Percy too.

Reply:
Working on that 👍

Reply:
I removed the random phrase functionality and instead I just loop through the different phrases in order so it always shows the same cards for users and for percy. When ready, we'll still be able to see the `same height` function in action and increasing the card count still works normally.